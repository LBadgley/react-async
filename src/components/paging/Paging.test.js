import React from 'react';
import { shallow } from 'enzyme';
import Paging from './Paging';

describe('paging component', () => {
  it('renders the paging component', () => {
    const wrapper = shallow(<Paging />);
    expect(wrapper).toMatchSnapshot();
  });
});
