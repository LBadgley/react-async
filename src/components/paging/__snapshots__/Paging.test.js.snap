// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`paging component renders the paging component 1`] = `
<Fragment>
  undefined / undefined
</Fragment>
`;
